src/app/component/login/login.component.scss

```diff
@@ -0,0 +1,21 @@
+
+.login-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100vh;
+
+    form {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: 1rem;
+
+     
+    }
+  
+    h3 {
```

Comment:
please give on element h3
class name and use in scss
its more common

Reply:
fixed